src/main/java/com/example/employee/model/Employee.java

```diff
@@ -19,6 +19,7 @@ public class Employee {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    private String personalId;
```

Comment:
Why don't we use just `id` we already have in this case?

Reply:
Because it is automatically generated and it is not required field to add new employee, so when the user like HR tries to add new employee in the system she/he do not indicate id, therefore we can't implement validation function and can't be able to avoid duplicated employees in the system.